<?xml version="1.0"?>
<launch>
    <arg name="Mono_image_topic" default="/camera/color/image_raw"/>
    <!--arg name="Mono_image_topic" default="/camera/rgb/image_color"/-->
    
    <arg name="path_to_vocabulary" default="/root/ORB_SLAM2/Vocabulary/ORBvoc.bin"/>
    <arg name="path_to_settings" default="/root/catkin_ws/src/orbslam-ros/launch/normal.yaml"/>
    <arg name="path_to_output" default="/mnt/d/WslSystem/share/dataset/output/normal/"/>
    <arg name="h_init_true" default="0.8"/>

    <node pkg="orbslam-ros" type="Mono" name="Mono_example" output="screen">
        <param name="Mono_image_topic" value="$(arg Mono_image_topic)"/>
        <param name="path_to_vocabulary" value="$(arg path_to_vocabulary)"/>
        <param name="path_to_settings" value="$(arg path_to_settings)"/>
        <param name="path_to_output" value="$(arg path_to_output)"/>
        <param name="h_init_true" value="$(arg h_init_true)"/>
    </node>

</launch>
